lib/fastlane/plugin/appcenter/actions/appcenter_upload_action.rb

```diff
@@ -258,6 +290,26 @@ def self.available_options
                                  is_string: false,
                              default_value: false),
 
+          FastlaneCore::ConfigItem.new(key: :mapping,
+                                  env_name: "APPCENTER_DISTRIBUTE_ANDROID_MAPPING",
+                               description: "Path to your Android mapping.txt",
+                                  optional: true,
+                                      type: String,
+                              verify_block: proc do |value|
+                                accepted_formats = [".txt"]
+                                if value
+                                  UI.user_error!("Couldn't find mapping.txt at path '#{value}'") unless File.exist?(value)
+                                  UI.user_error!("Only \"mapping.txt\" file name is allowed, you provided \"#{File.name(value)}\"") unless accepted_formats.include? File.extname(value)
```

Comment:
The error message is confusing. Anything that ends with `.txt` is allowed, right?

Reply:
For the first error I'd propose to neutralise with "Couldn't find mapping file at path". For the second it depends on what we do according to @ranterle comment.

Reply:
Turned out I didn't push that change. Already fixed messages. Second one is `Only \"*.txt\" formats are allowed, you provided ...` to be unified with other error messages